api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementService.java

```diff
@@ -114,4 +120,12 @@ private EligibilityStatus determineEligibilityStatus(EligibilityResponse respons
         return response.getEligibilityStatus();
     }
 
+    private QualifyingBenefitEligibilityStatus determineQualifyingBenefitStatus(EligibilityResponse response) {
```

Comment:
Would it make sense for this to be in the enum itself called something like `fromEligibilityResponse`? Would certainly be easier to unit test then.